chainercv/utils/download.py

```diff
@@ -21,10 +21,11 @@ def _reporthook(count, block_size, total_size):
         start_time = time.time()
         return
     duration = time.time() - start_time
-    if duration < 1e-4:
-        duration = 1e-4
     progress_size = int(count * block_size)
-    speed = int(progress_size / (1024 * duration))
+    if duration > 1e-4:
```

Comment:
How about using `try` and `except` to catch `ZeroDivisionError`. I think this is better because we can remove a magic value `1e-4`.